### PR TITLE
Error in scanvi_ref multiclass accuracy calculation

### PR DESCRIPTION
**Pull request by ori-kron-wis**
added -1 to the squeeze in order to deal with the case of dim==1 in a multiclass batch where there is 1 sample for the usage of scvi-> scanvi transfer learning

close https://github.com/scverse/scvi-tools/issues/2934

### CODE REVIEW

**Comment by canergen** [comment]
As far as I understand it. It's not really for the case dim==1 but for the case .shape =(1,1). It doesn't require a comment and needs a CHANGELOG entry.

File: CHANGELOG.md
Comment (canergen) [inline CHANGELOG.md:1]: Please refer to the class for the DataSplitter. And the message should be: 'Fix logging of accuracy for cases with 1 sample per class in scANVI'. I assume the transfer actually doesn't matter.

Comment (ori-kron-wis) [reply CHANGELOG.md:1]: changed

File: CHANGELOG.md [hunk 1]
```diff
@@ -97,6 +97,8 @@ to [Semantic Versioning]. Full commit history is available in the
 
 #### Fixed
 
+- Fix logging of accuracy for cases with 1 sample per class in scANVI {pr}`2938`.
+      the DataSplitter {pr}`2938`.
```
Comment (canergen) [inline CHANGELOG.md:101]:
```suggestion
```

**Pull request merged**